lib/index.js

```diff
@@ -1 +1,81 @@
 'use strict';
+
+const parseConfig = require('./config');
+const TestRepeater = require('./test-repeater');
+const {assertNonNegativeInteger} = require('./validators');
+const {logger} = require('./utils');
+
+module.exports = (hermione, opts) => {
+    const pluginConfig = parseConfig(opts);
+    let cliTool;
+
+    if (!pluginConfig.enabled || hermione.isWorker()) {
+        return;
+    }
+
+    hermione.on(hermione.events.CLI, (cli) => {
+        cliTool = cli;
+        cli.option(
+            '--repeat <number>',
+            'how many times should tests be repeated regardless of the result',
+            (value) => parseNonNegativeInteger(value, 'repeat')
+        );
+    });
+
+    hermione.on(hermione.events.INIT, () => {
+        const repeat = cliTool.repeat || pluginConfig.repeat;
+
+        if (!repeat) {
+            return;
+        }
+
+        const limitedRepeat = limitRepeatCount(repeat, pluginConfig.minRepeats, pluginConfig.maxRepeats);
+
+        if (limitedRepeat !== repeat) {
+            logger.info(`repeats count is changed from ${repeat} to ${limitedRepeat} since the specified value is out of limits`);
```

Comment:
repeat count was changed from ${repeat} to ${limitedRepeat} because it should be in range 10-50